lib/Parser.js

```diff
@@ -312,16 +312,19 @@ void OnParserTokenResult;
 
 function expected(
   expectedDescription /*: string */,
-  actual /*: any */
+  actual /*: mixed */
 ) /*: typeof ParseError */ {
   return {
     tag: 'ParseError',
-    message: `Expected ${expectedDescription} but got: ${JSON.stringify(
-      actual
-    )}`,
+    message: `Expected ${expectedDescription} but got: ${stringify(actual)}`,
   };
 }
 
+function stringify(json /*: mixed */) /*: string */ {
```

Comment:
Duplicated func definition?

Reply:
Yes! I’m not sure if it makes sense to have a shared definition.

Reply:
yeah not too worried about this duplication, is pretty simple function.